junifer/markers/collection.py

```diff
@@ -104,8 +119,8 @@ def validate(self, datagrabber) -> None:
 
         for marker in self._markers:
             logger.info(f"Validating Marker: {marker.name}")
-            m_data = marker.validate(t_data)
+            m_data = marker.validate_input(t_data)
```

Comment:
This should still be validate.

There are two methods:

`validate_input`: checks the input
`get_output_kind`: gives the output kind (given the input)

`validate` calls both of them

Reply:
Okay makes sense since it comes from `PipelineStepMixin`.